.github/workflows/ci.yml

```diff
@@ -7,13 +7,13 @@ on:
 jobs:
   sanity:
     name: sanity
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
Not sure we want to go this far. Could introduce instability without us realizing where it comes from. I'm ok with bumping it. But going flat-out latest I'm not a fan of in CI.

Reply:
I was thinking similar, but preflight was using this so I thought it would be okay. It looks like we have two choices.

- `ubuntu-20.04`
- `ubuntu-22.04` (which is `latest`)

based on this depreciation [article](https://github.blog/changelog/2022-08-09-github-actions-the-ubuntu-18-04-actions-runner-image-is-being-deprecated-and-will-be-removed-by-12-1-22/). Should we just go to the `20.04` for now?

Reply:
22.04 would be fine. Updating it once every two years would not be horrible.